src/Utilities/GameUtils.java

```diff
@@ -0,0 +1,90 @@
+package Utilities;
+
+import bricklets.Wall;
```

Comment:
Agreed.  I felt ugly submitting that portion.  I planned to make the entity class accept handlers for updates & rendering so you no longer need to extend entity for trivial things (or maybe never) and then I'll delete the Wall class.